src/services/authenticator.js

```diff
@@ -64,16 +64,24 @@ function Authenticator({
 
   this.logout = async (opts = {}) => {
     const basePath = env.TOKEN_PATH || os.homedir();
+
     const pathForestrc = `${basePath}/.forestrc`;
     const forestToken = this.getVerifiedToken(pathForestrc);
-    const pathLumberrc = `${basePath}/.lumberrc`;
-    const isLumberLoggedIn = this.getVerifiedToken(pathLumberrc);
-
     if (forestToken) {
       fs.unlinkSync(pathForestrc);
       await applicationTokenService.deleteApplicationToken(forestToken);
     }
+
+    const pathForestForestrc = `${basePath}/.forest.d/.forestrc`;
+    const forestForestToken = this.getVerifiedToken(pathForestForestrc);
+    if (forestForestToken) {
+      fs.unlinkSync(pathForestForestrc);
+      await applicationTokenService.deleteApplicationToken(forestForestToken);
+    }
```

Comment:
These lines are the same lines than 68-73 with only change is the path.
No time to factorize for now